components/Send/Send.tsx

```diff
@@ -88,15 +89,17 @@ const Send: React.FunctionComponent<SendProps> = ({
   const { decimalSeparator } = getNumberFormatSettings();
   // transparent is not spendable.
   const spendable = totalBalance.spendablePrivate + totalBalance.spendableOrchard;
-  const stillConfirming = parseFloat(spendable.toFixed(8)) !== totalBalance.total - totalBalance.transparentBal;
+
+  const stillConfirming =
+    parseFloat(spendable.toFixed(8)) !== totalBalance.total - (someUnconfirmed ? 0 : totalBalance.transparentBal);
   const showShieldInfo =
     totalBalance &&
-    totalBalance.transparentBal > 0 &&
-    totalBalance.transparentBal + totalBalance.privateBal > info.defaultFee;
+    (someUnconfirmed ? 0 : totalBalance.transparentBal) > 0 &&
+    (someUnconfirmed ? 0 : totalBalance.transparentBal) + totalBalance.spendablePrivate > info.defaultFee;
   const showUpgradeInfo =
     totalBalance &&
-    totalBalance.transparentBal <= 0 &&
-    totalBalance.transparentBal + totalBalance.privateBal > info.defaultFee;
+    (someUnconfirmed ? 0 : totalBalance.transparentBal) <= 0 &&
```

Comment:
if transparent balance is less than 0 !??

Reply:
Well, you know Just in case... who knows in the future what value this field can have...

Reply:
Can this field be negative? Isn't that an error?